### PR TITLE
docs: add SPDX license identifier

### PR DESCRIPTION
**Pull request by leogr**
**What type of PR is this?**
/kind cleanup
/kind documentation

**Any specific area of the project related to this PR?**

/area library
/area cli
/area tests
/area examples

**What this PR does / why we need it**:

See https://github.com/falcosecurity/evolution/issues/318

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: cmd/artifact/artifact.go [hunk 1]
```diff
@@ -1,4 +1,5 @@
-// Copyright 2022 The Falco Authors
+// SPDX-License-Identifier: Apache-2.0
```
[the remark this answers is not in the document]
Comment (leogr) [reply cmd/artifact/artifact.go:1]: Good catch. Fixing.

[the remark this answers is not in the document]
Comment (leogr) [reply cmd/artifact/artifact.go:1]: Fixed!

**Review by FedeDP — APPROVED** [review]
/approve

**Pull request merged**